### PR TITLE
Integration test framework

### PR DESCRIPTION
**Pull request by tommydeboer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,3 +57,36 @@ pip install -e .
...
+By default the localhost is chosen (username: admin, password: admin), but it's possible to configure a different server:
+
+```
+python setup.py test --addopts "-m integration --ignore=tests/unit/ --url=<your_url> --username=<admins username> --password=<admins password>" 
+```
+
+#### Running tests in PyCharm
+To run the tests in PyCharm, first set the default test runner to 'pytest'. 
+
+![Configure default test runner](docs/default_test_runner.png)
```
Comment (dennishendriksen) [inline README.md]: Already set to pytest in a fresh PyCharm install on my system.

File: README.md [hunk 2]
```diff
@@ -57,3 +57,36 @@ pip install -e .
...
+
+Then create pytest run configurations using the same arguments as above:
+
+![Setting up run configuration](docs/run_configuration.png)
```
Comment (dennishendriksen) [inline README.md]:
```C:\Users\Dennis\Dev\molgenis-tools-commander\env\Scripts\python.exe "C:\Program Files\JetBrains\PyCharm 2018.3.4\helpers\pycharm\_jb_unittest_runner.py" --path C:/Users/Dennis/Dev/molgenis-tools-commander -- -m unit --ignore=tests/integration/
usage: python -m unittest discover [-h] [-v] [-q] [--locals] [-f] [-c]
                                   [-k TESTNAMEPATTERNS] [-s START]
                                   [-p PATTERN] [-t TOP]
python -m unittest discover: error: unrecognized arguments: -m --ignore=tests/integration/
Launching unittests with arguments python -m unittest discover -s C:/Users/Dennis/Dev/molgenis-tools-commander -t C:\Users\Dennis\Dev\molgenis-tools-commander -m unit --ignore=tests/integration/ in C:\Users\Dennis\Dev\molgenis-tools-commander


Process finished with exit code 2

File: Jenkinsfile [hunk 3]
```diff
@@ -33,7 +33,7 @@ pipeline {
             steps {
                 container('python') {
                     sh "pip install ."
-                    sh "python setup.py test"
+                    sh "python setup.py test --addopts '-m unit --ignore=tests/integration/'"
```
[the remark this answers is not in the document]
Comment (tommydeboer) [reply Jenkinsfile]: Known issue: https://github.com/molgenis/molgenis-tools-commander/issues/50, should be fixed separately

File: README.md [hunk 4]
```diff
@@ -57,3 +57,36 @@ pip install -e .
...
+To run all the unit tests from the command line, go:
+
+```
+python setup.py test --addopts "-m unit --ignore=tests/integration/" 
+```
+
+The integration tests require a running MOLGENIS application to test against. They can be run like this:
+ 
+```
+python setup.py test --addopts "-m integration --ignore=tests/unit/"
```
[the remark this answers is not in the document]
Comment (tommydeboer) [reply README.md]: See previous comment

File: README.md (hunk 1 above)
Comment (tommydeboer) [reply README.md]: I'm not sure if it's the same for everybody and it can be a gotcha so I suggest to keep it in there.

File: README.md (hunk 2 above)
Comment (tommydeboer) [reply README.md]: I think you missed an argument value, it should say:

`-m unit --ignore=tests/integration/`

Comment (dennishendriksen) [reply README.md]: The argument was ok. Turns out I had to create a 'Python test' of type 'pytest' instead of unit test.

**Pull request merged**